kinesis-produce-consume/README.md

```diff
@@ -0,0 +1,57 @@
+Spring Cloud Stream Kinesis Sample
+=============================
+
+In this *Spring Cloud Stream* sample, a Controller exists which can receive POST requests containing Order objects. These Orders are added to a Kinesis Stream. The sample application consumes the Orders from the stream.
+
+## Requirements
+
+To run this sample, you will need to have installed:
+
+* Java 8 or Above
+
+This example requires and AWS account, it will create a Kinesis stream which will be chargeable. The AWS credentials will be obtained using the default credential provider chain. 
+
+## Code Tour
+
+This sample is a Spring Boot application that uses Spring Cloud Stream to produce and consume data to a Kinesis Stream. The sample has the following components:
+
+* KinesisApplication - The Spring Boot Main Application
+* OrderController - The Controller exposing POST and GET endpoints 
+* OrderRepository - An in memory database for storing Order objects
```

Comment:
Why do we need this functionality if it isn't important for this kind of sample?
I just feel confused a bit, when I would like to take a look to the Kinesis variant of the SCSt, but I meet there something about JPA, more over with an in-memory DB.
Can't we simplify the sample do not scary end-users like me, who doesn't know im-memory JPA?
Thanks